{"name": "Проверка невозможности изменения данных неавторизованным пользователем", "status": "broken", "statusDetails": {"message": "NameError: name 'burger_api' is not defined", "trace": "self = <tests.test_changing_user_data.TestChangeUserData object at 0x000001DA710E2BA0>, key = 'email', value = ' vladvladov@yandex.ru '\n\n    @allure.title(\"Проверка невозможности изменения данных неавторизованным пользователем\")\n    @allure.description(\n        \"Невозможность редактирования данных неавторизованным: email, name, password\")\n    @pytest.mark.parametrize(\"key, value\",\n                             [\n                                 (\"email\", \" vladvladov@yandex.ru \"),\n                                 (\"name\", \" Vlad Vladov \"),\n                                 (\"password\", \"777777\")\n                             ])\n    def test_change_not_auth_user_data_fail(self, key, value):\n>       user_data = burger_api.create_user_body()\nE       NameError: name 'burger_api' is not defined\n\ntests\\test_changing_user_data.py:41: NameError"}, "description": "Невозможность редактирования данных неавторизованным: email, name, password", "parameters": [{"name": "key", "value": "'email'"}, {"name": "value", "value": "' vladvladov@yandex.ru '"}], "start": 1737281394895, "stop": 1737281394895, "uuid": "0b813c9f-d607-4fa4-85fc-319231bf1e51", "historyId": "8bb9d1921054176e3af82e22782ada70", "testCaseId": "35930caa214894d7f1af0597191b315c", "fullName": "tests.test_changing_user_data.TestChangeUserData#test_change_not_auth_user_data_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_changing_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "01ms32"}, {"name": "thread", "value": "10576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_changing_user_data"}]}